filemanager/digitaloceanmanager.go

```diff
@@ -290,7 +290,7 @@ type digitalOceanListSession struct {
 func (l *digitalOceanListSession) Next() (fileObjects []*FileInfo, err error) {
 	manager := l.manager
 	if !l.isTruncated {
-		manager.logger.Infof("Manager is truncated: %v so returning here", l.isTruncated)
+		manager.logger.Debugf("Manager is truncated: %v so returning here", l.isTruncated)
```

Comment:
When possible we should try to use the structured non-sugared logger, i.e. `manager.logger.Debugn`.